all.json

```
@@ -27,7 +27,9 @@
 	],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"walletconnect-secured.crypto247network.com",
```

```suggestion
		"walletconnect-secured.crypto247network.com",
		"crypto247network.com",
```

Also marked unsafe on Google.